projects/demo/src/modules/components/action-bar/examples/3/index.html

```diff
@@ -0,0 +1,35 @@
+<button
+    automation-id="open-table-bar-on-top"
+    size="m"
+    tuiButton
+    type="button"
+    (click)="open.set(true)"
+>
+    Show TableBar on top
+</button>
+
+<tui-action-bar
+    *tuiActionBar="open()"
+    size="s"
+>
+    <span [style.width.%]="isMobile() ? 100 : null">Table bar opened</span>
+
+    <button
+        appearance="glass"
+        iconStart="@tui.trash"
+        tuiButton
+        type="button"
+        (click)="open.set(false)"
+    >
+        Remove
+    </button>
+    <button
+        iconStart="@tui.x"
+        tuiIconButton
+        type="button"
+        [appearance]="isMobile() ? 'glass' : 'icon'"
```

Comment:
Maybe we should simplify the example since it should focus on positioning, rather than on content?